docs/version0.17.md

```diff
@@ -47,14 +48,20 @@ OpenJ9 release 0.17.0 supports OpenJDK 8, 11, and 13. Binaries are available fro
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
-### New shared classes cache suboption
+### New shared classes cache suboptions for layered caches
 
-When creating a persistent shared classes cache, the OpenJ9 VM checks that there is sufficient disk space available on the file
-system. For file systems that do not support the checking of free space, you can set the `-Xshareclasses:noPersistentDiskSpaceCheck` option, which causes the VM to skip the space checking operation. If there isn't enough disk space available when the cache is written, a **SIGBUS** or **SIGSEGV** signal occurs and the VM ends. For more information, see the [-Xshareclasses:noPersistentDiskSpaceCheck](xshareclasses.md#nopersistentdiskspacecheck) option.
+**(Experimental, 64-bit only)**
 
-### Digest algorithm is re-enabled
+Two new suboptions, `createLayer` and `layer=<number>`, are available for creating layered caches, where a cache builds on another cache with the same name. You can use these suboptions to save space when building a Docker container, for example. For more information, see [`-Xshareclasses:layer`](xshareclasses.md#layer).
 
-Issue [#5611](https://github.com/eclipse/openj9/issues/5611) is fixed, so support for the Digest algorithm is re-enabled. For more information about this support, see [Cryptographic operations]( introduction.md#cryptographic-operations).
+A new suboption, [`destroyAllLayers`](xshareclasses.md#destroyalllayers), is available for destroying all the layers of a layered cache.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Because these suboptions are experimental, do not use them in a production environment.
+
+### New shared classes cache suboption to skip disk space check
+
+When creating a persistent shared classes cache, the OpenJ9 VM checks that there is sufficient disk space available on the file
+system. For file systems that do not support the checking of free space, you can set the `-Xshareclasses:noPersistentDiskSpaceCheck` option, which causes the VM to skip the space checking operation. If there isn't enough disk space available when the cache is written, a **SIGBUG** or **SIGSEGV** signal occurs and the VM ends. For more information, see the [-Xshareclasses:noPersistentDiskSpaceCheck](xshareclasses.md#nopersistentdiskspacecheck) option.
```

Comment:
This is going to revert the fix of SIGBUG -> SIGBUS

Reply:
Well-spotted - bad merge conflict resolution :(  I have no idea how you undo such things, but it's only a typo so I've just changed it again in this PR.